crates/ursa-gateway/src/config.rs

```diff
@@ -96,6 +103,10 @@ impl Default for GatewayConfig {
                  * mh_url: "https://cid.contact/multihash".into(),
                  */
             },
+            cache: CacheConfig {
+                max_size: 200_000_000,  // 200MB
```

Comment:
maybe i missed but is this cache for content or index list?

Reply:
It's content cache.

Reply:
200MB is enough for content cache?

Reply:
https://github.com/fleek-network/ursa/blob/b1d6cd56911ac5174fc4c1126cf5cb27c5838408/infra/ursa/data/nginx/app.conf#L1

nginx on each node has 10GB

Reply:
Sure, 10GB is not feasible for a development machine, it's just a default configuration for local development.
Let's come up with the configuration mode (dev, test, prod), sound good?

Reply:
I mean on the gateway it could be even higher right, like even 3x that. Yeah lets develop for prod env. The simplest case is to make it variable and not const

Reply:
Sure, that is a configuration variable, subject to change via `config.toml` or `cli argument`.